packages/mui-system/package.json

```
@@ -41,8 +41,8 @@
   "peerDependencies": {
     "@emotion/react": "^11.5.0",
     "@emotion/styled": "^11.3.0",
-    "@types/react": "^16.8.6 || ^17.0.0",
-    "react": "^17.0.0"
+    "@types/react": "^16.8.6 || ^17.0.0 || ^18.0.0",
```

@michaldudak Should we remove the support for `^16.8.6`?

```suggestion
    "@types/react": "^17.0.0 || ^18.0.0",
```

For instance https://unpkg.com/browse/@types/react@16.9.56/index.d.ts says

> Type definitions for React 16.9

---

I believe so, yes.